packages/suite/src/constants/suite/staking.ts

```diff
@@ -0,0 +1,3 @@
+import { NetworkSymbol } from '@suite-common/wallet-config';
+
+export const STAKE_SYMBOLS: NetworkSymbol[] = ['eth', 'tgor', 'thol'];
```

Comment:
This constant can be derived from `networks` in `networksConfig.ts` `features` with `networkType` `ethereum`

Reply:
https://github.com/everstake/trezor-suite/commit/9d06b0a8da0dc44f40d19f7206507188b57f0e19